crates/core/src/ring.rs

```diff
@@ -603,6 +986,14 @@ impl Display for Distance {
     }
 }
 
+impl std::ops::Sub<Distance> for Distance {
+    type Output = f64;
+
+    fn sub(self, rhs: Distance) -> Self::Output {
+        self.0 - rhs.0
```

Comment:
Could this result in a negative distance?

Reply:
Yes, however this is not needed anymore so I am going to remove this completely.